pallets/collator-assignment/src/lib.rs

```diff
@@ -104,6 +126,13 @@ pub mod pallet {
     pub(crate) type PendingCollatorContainerChain<T: Config> =
         StorageValue<_, Option<AssignedCollators<T::AccountId>>, ValueQuery>;
 
+    /// Randomness from previous block. Used to shuffle collators on session change.
+    /// Should only be set on the last block of each session and should be killed on the on_initialize of the next block.
+    /// The default value of [0; 32] disables randomness in the pallet.
+    #[pallet::storage]
```

Comment:
While I like this, I am also missing a way to disable it. This probably cannot serve as a kill switch, as it is set  on_finalize of the last block before the session, so we would need to set it  0 on that block.

I think we can have an additional storage that serves as s kill switch, any opposition for this?